optimize/self-managed/optimize-deployment/migration-update/3.10-to-3.11.md

```diff
@@ -29,3 +29,51 @@ If you need to update your Elasticsearch cluster, refer to the general [Elastics
 Prior to Optimize 3.11.0, Optimize has performed collection role cleanup after syncing identities with the engine. From
 Optimize 3.11.0 onwards, this is now disabled by default. It can be reenabled by setting the
 `import.identitySync.collectionRoleCleanupEnabled` property value to `true`
+
+# API changes
+
+Structure of raw data reports is now changed. The raw data report properties named `numberOfIncidents`, `numberOfOpenIncidents` and `numberOfUserTasks` are now renamed to `incidents`, `openIncidents` and `userTasks` respectively and grouped together in a single property named `counts`.
```

Comment:
```suggestion
The data structure of raw data reports is now changed. The raw data report properties named `numberOfIncidents`, `numberOfOpenIncidents` and `numberOfUserTasks` are now renamed to `incidents`, `openIncidents` and `userTasks` respectively and grouped together in a single property named `counts`.
```
Also, won't they have the `count:prefix` too?

I think that we should be explicit in why this is important, which in this case is because it changes the response of the JSON and CSV export right?